modules/transports/optional/fix/src/test/java/org/apache/synapse/transport/fix/FIXTransportListenerInitTest.java

```diff
@@ -0,0 +1,49 @@
+package org.apache.synapse.transport.fix;
+
+import junit.framework.TestCase;
+import org.apache.axis2.context.ConfigurationContext;
+import org.apache.axis2.description.AxisService;
+import org.apache.axis2.description.Parameter;
+import org.apache.axis2.description.TransportInDescription;
+import org.apache.axis2.description.TransportOutDescription;
+import org.apache.axis2.engine.AxisConfiguration;
+import org.apache.axis2.transport.testkit.axis2.TransportDescriptionFactory;
+import org.junit.Assert;
+import org.testng.annotations.Test;
+
+public class FIXTransportListenerInitTest extends TestCase {
```

Comment:
Test class name should be same as the class you are going to test and suffix with Test. Ex: FIXTransportListenerTest.java. All the method comes under the test class

Reply:
done